src/main/kotlin/Operator.kt

```diff
@@ -0,0 +1,17 @@
+import java.lang.IllegalArgumentException
+
+enum class Operator(val value: String) {
+    ADD("+"),
+    MINUS("-"),
+    TIMES("*"),
+    DIV("/");
+
+    companion object {
+        fun of(s: String): Operator {
+            values().forEach {
+                if (it.value == s) return it
+            }
+            throw IllegalArgumentException("적절한 연산자를 찾을 수 없습니다.")
```

Comment:
```suggestion
         return values().firstOrNull { it.value == s } ?: throw IllegalArgumentException("적절한 연산자를 찾을 수 없습니다.")
```

`firstOrNull` 을 이용하면 간결하게 표현이 가능합니다. 😊

링크
- [[Android/안드로이드] first, firstOrNull, last, lastOrNull 이건 뭐지?](https://solution94.tistory.com/104) https://solution94.tistory.com/104